CONTRIBUTING.md

```diff
@@ -33,12 +33,12 @@ on the new feature or improvement.
 
 ### Maven tips
 
-A `./mvnw clean install` runs basically "everything" except release/deployment stuff. This is often
+A `./mvnw --threads 1C clean install` runs basically "everything" except release/deployment stuff. This is often
```

Comment:
i found no maven config setting to apply this universally, so it has to be added everywhere it seems